src/Compilers/Server/VBCSCompiler/NamedPipeClientConnection.cs

```diff
@@ -5,270 +5,84 @@
 using Roslyn.Utilities;
 using System;
 using System.Diagnostics;
-using System.Globalization;
 using System.IO;
+using System.IO.Pipes;
+using System.Runtime.CompilerServices;
+using System.Security.Principal;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.CommandLine;
-using System.IO.Pipes;
+using System.Security.AccessControl;
+
+#nullable enable
```

Comment:
When reviewing this file best to look at the new contents. The old file used to be a mix of named pipe logic + server logic. The server logic is now inside `ClientConnectionHandler` and the resulting file is just about managing the named pipe.